src/selfplay/tournament.cc

```diff
@@ -233,8 +237,13 @@ void SelfPlayTournament::PlayOneGame(int game_number) {
   }
   auto& game = **game_iter;
 
+  bool use_resign = true;
+  if (kResignPlaythrough > 0) {
```

Comment:
`if` is not really needed. (and creation of a game is a rare event so we don't have to care about performance)